mos_tests/neutron/python_tests/test_dhcp_agent.py

```diff
@@ -55,15 +49,40 @@ def prepare_openstack(self, setup):
                                         network_id=ext_network['id'])
         logger.info('router {} was created'.format(self.router['id']))
 
-    def get_all_dhcp_agents(self):
-        dhcp_agts = [agt for agt in self.os_conn.neutron.list_agents(
-                                       binary='neutron-dhcp-agent')['agents']]
-        return dhcp_agts
+        self.dhcp_agent_ids = [agt['id'] for agt in
+                                self.os_conn.neutron.list_agents(
```

Comment:
All checks are passed so far with that space. Did not find any problems with manual flake8 execution as well.